plugins/mqtt/package.json

```diff
@@ -29,17 +29,18 @@
       ]
    },
    "dependencies": {
-      "@types/node": "^16.6.1",
       "aedes": "^0.46.1",
-      "axios": "^0.23.0",
+      "axios": "^1.6.7",
+      "color-convert": "^2.0.1",
```

Comment:
I'd rather not introduce a dependency for a simple function here: from copilot: 

```ts
function rgbToHsv(r: number, g: number, b: number): [number, number, number] {
    r /= 255, g /= 255, b /= 255;

    let max = Math.max(r, g, b), min = Math.min(r, g, b);
    let h, s, v = max;

    let d = max - min;
    s = max == 0 ? 0 : d / max;

    if (max == min) {
        h = 0; // achromatic
    } else {
        switch (max) {
            case r: h = (g - b) / d + (g < b ? 6 : 0); break;
            case g: h = (b - r) / d + 2; break;
            case b: h = (r - g) / d + 4; break;
        }
        h /= 6;
    }

    return [h, s, v];
}
```

Reply:
this dependency looks relatively small and 1 level deep, so this could be fine. I generally prefer avoiding that on trivial stuff like this.

Reply:
I have completed this item